scraping/scraping/spiders/_base.py

```diff
@@ -131,14 +129,21 @@ def parse_PRODUCT(self, response: Union[SplashJsonResponse, ScrapyHttpResponse])
         Args:
             response (SplashJsonResponse): Response from a performed request
         """
+
+        request_meta_information = response.meta.get("amazon_price", None)
```

Comment:
I still think we should not use shop specific things here. Since this is the base implementation, we should be as generic as possible here. Let's use: `response.meta.get("request_meta_information", None)`, where `request_meta_information` is a dictionary.